CREATE TABLE t2 (cc int);
NOTICE:  00000: test_event_trigger: ddl_command_start CREATE TABLE
ERROR:  42P07: relation "t2" already exists
